recipes/scripting/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "scripting" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/csdms/py-scripting/archive/v{{ version }}.tar.gz
+  fn: py-scripting-{{ version }}.tar.gz
+  sha256: 7ff93a9af586c93a41808f5132f75dc58c9a383abe702c5e5a87cc95f9585670
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - setuptools
+    - python
+    - pip
+    - pytest
```

Comment:
Do you really need ``pytest`` as dependency for this project or just for test it?